source/code-generation/query-converter/convert-triggers.txt

```diff
@@ -0,0 +1,202 @@
+.. _rm-convert-triggers:
+
+================
+Convert Triggers
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can import and convert your SQL triggers to 
+:ref:`MongoDB Atlas Triggers <atlas-triggers>` with the query converter. 
+The query converter considers the SQL code and relational schema defined 
+in your project when converting your triggers.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
+
+Before you Begin
+----------------
+
+- Your relational database must have at least one trigger defined in order
+  to use query converter trigger conversion.
+
+- .. include:: /includes/fact-query-converter-disclaimer.rst
```

Comment:
I agree this could be a warning, but do to the fact that there are only 2 bullet points in the section currently I will elect to keep this as is. I will consider changing in future iterations if we add more BYB points.